src/pretix/control/urls.py

```diff
@@ -303,4 +304,5 @@
         url(r'^checkinlists/(?P<list>\d+)/delete$', checkin.CheckinListDelete.as_view(),
             name='event.orders.checkinlists.delete'),
     ])),
+    url(r'^event/(?P<organizer>[^/]+)', RedirectView.as_view(pattern_name='organizer')),
```

Comment:
This doesn't seem like optimal behaviour here: regexes can be notoriously long-running, and additionally `(?P<organizer>[^/]+)` catches more than you'd like! What you are probably looking for is this:

```suggestion
    path('event/uberspace', RedirectView.as_view('/control/organizer/uberspace/')),
```

Reply:
(SCNR, happy holidays everybody 🎅)